slim_trees/lgbm_booster.py

```diff
@@ -183,9 +225,26 @@ def _decompress_booster_handle(compressed_state: Tuple[str, List[dict], str]) ->
         tree_str += "\ninternal_weight=" + ("0 " * num_nodes)[:-1]
         tree_str += "\ninternal_count=" + ("0 " * num_nodes)[:-1]
         tree_str += f"\nis_linear={tree['is_linear']}"
-        tree_str += f"\nshrinkage={tree['shrinkage']}"
-        tree_str += "\n\n\n"
+        if is_linear:
+            # TODO: attributes: leaf_features, leaf_coeff, leaf_const, num_features
+            tree_str += f"\nleaf_const={' '.join(str(x) for x in tree['leaf_const'])}"
```

Comment:
If it's not terribly inefficient I think it would be more readable like this:

```py
tree_str += "\nleaf_const=" + " ".join(map(str, tree["leaf_const"]))
```